packages/metal-quartz-link/src/MetalQuartzLink.js

```diff
@@ -0,0 +1,104 @@
+'use strict';
+
+import Component from 'metal-component';
+import MetalQuartzIcon from 'metal-quartz-icon';
+import Soy from 'metal-soy';
+import { core, object } from 'metal';
+import { validators } from 'metal-state';
+
+import templates from './MetalQuartzLink.soy.js';
+
+/**
+ * Implementation of the Metal Quartz Link.
+ */
+class MetalQuartzLink extends Component {
+	/**
+	 * @inheritDoc
+	 */
+	created() {
+		MetalQuartzLink.instances.push(this);
```

Comment:
It is, one just needs to add the `ref` attribute to the components that should be exposed via the `this.components` API.

See https://github.com/metal/metal.js/blob/master/packages/metal-incremental-dom/src/render/render.js#L159-L165